ystia/yorc
# Feature/gh 689 a4c 3

# Pull Request description

## Description of the change

### What I did

`versions.yaml`
Updated Alien4Cloud version

`tosca_types.zip`
Updated Alien4Cloud types

### How to verify it

Bottstrap yorc, keeping bootstrap default value for Alien4Cloud URL.
Check the bootrapped setup runs Alien4Cloud 3.0.0

### Description for the changelog

Support Alien4Cloud 3.0 ([GH-689](https://github.com/ystia/yorc/issues/689))

## Applicable Issues

Closes #689
